internal/juju/offers.go

```diff
@@ -310,7 +310,16 @@ func (c offersClient) ConsumeRemoteOffer(input *ConsumeRemoteOfferInput) (*Consu
 
 	localName, err := client.Consume(consumeArgs)
 	if err != nil {
-		return nil, err
+		// Check if SAAS is already created. If so return offer response instead of error
+		if strings.Contains(err.Error(), "saas application already exists") {
```

Comment:
This error should be of type jujuerrors.AlreadyExists, please use the error type instead of a string for matching. String matching should be the last resort if types do not exist.

`if jujuerrors.Is(err, jujuerrors.AlreadyExists) {`

The import is `jujuerrors "github.com/juju/errors"`

Reply:
Done